### ISSUE TITLE
Variable substitution does not work inside request bodies

### ISSUE THREAD

**Issue by DieterHolvoet** [httpyac/httpyac.github.io#36]
Example:
```
POST https://{{host}}/oauth/token
Content-Type: application/x-www-form-urlencoded

grant_type=client_credentials&client_id={{clientId}}&client_secret={{clientSecret}}

> {% client.global.set('accessToken', response.body.access_token); %}
```

`.vscode/settings.json`
```json
{
    "httpyac.environmentVariables": {
        "$shared": {
            "host": "some-host.test",
            "clientId": "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX",
            "clientSecret": "XXXXXXXXXXXXXX"
        }
    }
}
```

`{{clientId}}` and `{{clientSecret}}` are not substituted, while `{{host}}` is.

**Comment by DieterHolvoet** [comment]
My bad, wrong repo.

**Comment by AnWeber** [comment]
@DieterHolvoet would also have been ok, with it. i'll look at it

**Comment by AnWeber** [comment]
The reason is that the body is encoded before replacing the variables and thus the brackets `{` are changed to `%7b`. You have the following workaround options:
-  remove header content-type, because this triggers the encoding
- use OAuth2 implmentation (see https://httpyac.github.io/guide/variables.html#oauth2-openid-connect)
- set body in a custom script , but not Intellij compatible

```
{{
  request.body = `grant_type=client_credentials&client_id=${clientId}&client_secret=${clientSecret}`
}}

POST https://{{host}}/oauth/token
Content-Type: application/x-www-form-urlencoded

> {% client.global.set('accessToken', response.body.access_token); %}
```

**Comment by DieterHolvoet** [comment]
Thanks for the quick response and the suggestions! None of these workarounds work for me, because I need Intellij compatibility and removing the `Content-Type` header breaks the request. I see you committed a fix, I'll be looking forward to the next release.

**Comment by AnWeber** [comment]
Fix is published with v5.2.1
